sql/create_population.sql

```diff
@@ -0,0 +1,67 @@
+CREATE OR REPLACE TABLE dcard-production.ad_ctr.ad_ctr_raw_population AS
+WITH
+deduplicated_ad_displayed AS (
+  SELECT
+    *,
+    ROW_NUMBER() OVER (PARTITION BY request_id ORDER BY arrived_at) AS r -- select first ad_display data
+  FROM
+    `dcard-production.bilanx_dimensionalized.ad_displayed`
+  WHERE
+    event_created_at_taipei_zone_date >= "{{TRAIN_START}}"
+    AND event_created_at_taipei_zone_date <= "{{TEST_END}}"
+    AND ad_type = 'smb'
+    AND request_id != ""
+    AND request_id IS NOT NULL
+),
+ad_displayed_data AS (
+  SELECT
+    *
+  FROM
+    deduplicated_ad_displayed
+  WHERE
+    r = 1
+),
```

Comment:
```suggestion
  HAVING r = 1
),
```
不確定能不能這樣，如果可以的話下面用到的 ad_displayed_data 要改名

Reply:
看起來不行 ＱＱ
```
Column r contains an analytic function, which is not allowed in HAVING clause at 
```